examples/ex19.cpp

```diff
@@ -0,0 +1,239 @@
+#include "mfem.hpp"
```

Comment:
I'll be happy to change the numbers and add headers in the style of the other MFEM examples.

I'm not sure what you mean by the LLNL copyright banner.  I don't see this in the other examples that I glanced at.  Should we add this to all of the examples?